### PR TITLE
added headers for your photos and favs

### PR DESCRIPTION
**Pull request by jakobroehrl**
Signed-off-by: Jakob <jakob.roehrl@web.de>
fixes: #114

### CODE REVIEW

File: src/router/index.js [hunk 1]
```diff
@@ -47,6 +47,11 @@ export default new Router({
 			path: '/',
 			component: Timeline,
 			name: 'root',
+			props: route => ({
+				path: `/${route.params.path ? route.params.path : ''}`,
```
Comment (skjnldsv) [inline src/router/index.js]: There is no path in the timeline :)
So it doesn't make sense, also isRoot should be always true then ?

Comment (Mikescops) [reply src/router/index.js]: True!

File: src/router/index.js [hunk 2]
```diff
@@ -77,9 +80,12 @@ export default new Router({
 			path: '/favorites',
 			component: Timeline,
 			name: 'favorites',
-			props: {
+			props: route => ({
+				path: `/${route.params.path ? route.params.path : ''}`,
```
Comment (skjnldsv) [inline src/router/index.js]: same goes for favs? :)

**Comment by Mikescops** [comment]
@jakobroehrl don't forget to lint your code, otherwise you don't pass the CI

**Comment by jakobroehrl** [comment]
> 
> 
> @jakobroehrl don't forget to lint your code, otherwise you don't pass the CI

Done, I hope we can merge now.

File: src/views/Timeline.vue [hunk 3]
```diff
@@ -33,12 +33,20 @@
 	</EmptyContent>
 
 	<!-- Folder content -->
-	<VirtualGrid v-else-if="!loading"
+	<div v-else-if="!loading">
+		<Navigation 
+		key="navigation"
```
Comment (skjnldsv) [inline src/views/Timeline.vue]: Indentation?

Comment (jakobroehrl) [reply src/views/Timeline.vue]: better now?

**Comment by skjnldsv** [comment]
Backport in https://github.com/nextcloud/photos/pull/172

**Pull request merged**